contrib/sync-places.py

```diff
@@ -63,34 +63,50 @@ async def do_sync(session, args):
 
         for name in config["places"]:
             matches = config["places"][name].get("matches", [])
+            matches = [tuple(m) if isinstance(m, list) else m for m in matches]
```

Comment:
I don't like the mixed types in the `matches` variable as it makes iterating a lot harder. It would be better if all entries in the `matches` array were a tuple of `(match_expression, name)` and `name` can just be `None` if there is no name for the match. This also works nicely because passing `None` to `add_place_match` means "don't rename"

So, with that in mind and the comment below, perhaps something like:

```python
matches = []
for m in config["places"][name].get("matches", []):
    if isinstance(m, dict):
        name = m.keys()[0]
        matches.append((m[name], name))
    else:
        matches.append((m, None))

```

Reply:
I've updated the PR accordingly.